dailyperf/tests/icon_border/test.md

```diff
@@ -13,28 +13,27 @@ _Between border and icon, witch one impact the most the performance?_
 |  N    |  Y    |  N   |   N   |
 
 ## The Question
-Today we got a good visual question, we have to choose between two designs to render events in a grid.
 
-The user must know what is the type of these events, so we have to choose between:
+Today we were offered two visual designs to render events in a grid, and let the user know what he's dealing with.
+The event type could be defined with:
 
- 1. Using a glyph to set the type of the event.
- 2. Using a border color to define the type of the event. 
+ 1. An icon
+ 2. A border color
 
 ## The answer
 
-Between the two choice we have a little performance difference. Border is faster than icon of 20ms in average.
+There's only a little performance difference between the two options. Border is faster than icon by 20ms in average.
 Following the [RAIL performance model](https://developers.google.com/web/tools/profile-performance/evaluate-performance/rail#animation-render-frames-every-16ms) we have a budget of 16 ms for animation.
-In our case, if we have to dynamically generate these events during an animation or an user input (scroll for example), our budget for a fluid animation (16ms) is already burn
-when using icons.
+In our case, dynamically generate these events during an animation or a user input (e.g. scroll) would already use up our all the budget for a fluid animation (16ms).
```

Comment:
"use up our all the budget for a fluid animation"? 

"use up all our budget for a fluide animation" no?